### DISCUSSION TITLE
Dados do INMET e do CPTEC

### DISCUSSION THREAD

**Discussion by jlucas577** [BrasilAPI/BrasilAPI#229] [Ideas]
Algumas instituições governamentais, como a Defesa Civil, necessitam utilizar em sua operação diaria dados do INMET e do CPTEC, seria um puta avanço ter isso tudo em uma API. Em um sistema que desenvolvi, precisei fazer uma gambiarra usando CURL para pegar capturar esses dados e conseguir trabalhar com eles.

**Comment by LucasMallmann** [comment]
Acho isso bem válido. Já trabalhei no CPTEC e realmente eu já sofria com a disposição de dados!

**Comment by RodriAndreotti** [comment]
Olá, pessoal!
Tudo bem?
Estive dando uma olhada nos dois órgãos e ambos parecem ter uma API já (Bem, o CPTEC, parece ser um webservice com retorno em XML, mas também tem).

Talvez possamos inserir um tratamento para estes dados aqui.

**Comment by RodriAndreotti** [comment]
Pessoal, comecei a fazer algumas rotinas nos web-services do CPTEC, como muitos são dependentes dos códigos de localização deles, comecei justamente pelo endpoint de cidades, abri uma discussão sobre aqui: https://github.com/BrasilAPI/BrasilAPI/discussions/172, quem quizer dar um pitaco, fique à vontade.

Aliás, @lucianopf, se quiser designar tbm a mim a issue, creio que consigo tratar todos os endpoints do cptec.
